### DISCUSSION TITLE
Module "PaperMod" is not compatible with this Hugo version

### DISCUSSION THREAD

**Discussion by mvrck96** [adityatelange/hugo-PaperMod#615] [Q&A]
Salut !

I'm running into WARN mesage when starting hugo server with PaperMod theme. I'm using Ubuntu 18.04, hugo version is v0.88.1, PaperMod cloned from repo. Termianl output:

```bash
WARN 2021/10/31 12:18:34 Module "PaperMod" is not compatible with this Hugo version; 
run "hugo mod graph" for more information.
```

`Hugo mod graph` output:

```bash
WARN 2021/10/31 12:24:47 Module "PaperMod" is not compatible with this Hugo version; 
run "hugo mod graph" for more information.
project PaperMod
```

**What should I do to remove this warning or this message is affecting nothing and I'm good to go with it ?**

**Comment by cashpw** [comment]
I encountered this same error when using the version of `hugo` provided by `apt` in Debian 10 (0.80). The error went away after installing the [latest version of Hugo from deb](https://github.com/gohugoio/hugo/releases).

**Comment by liuchaowen** [comment]
`
ignoreErrors: ["error-remote-getjson"]  #yml
or
ignoreErrors = ["error-remote-getjson"]  #toml 
` 

add this to your config file
